libraries/libfc/include/fc/variant.hpp

```diff
@@ -694,6 +695,35 @@ namespace fc
       n = boost::multiprecision::number<T>(v.get_string());
    }
 
+   template<typename T> void to_variant( const boost::dynamic_bitset<T>& bs, fc::variant& v ) {
+      auto num_blocks = bs.num_blocks();
+      if( num_blocks > MAX_NUM_ARRAY_ELEMENTS ) throw std::range_error( "too large" );
```

Comment:
Use `FC_ASSERT`

Reply:
Cannot use `FC_ASSERT` in `variant.hpp` and similar files, as there is a circular includes from `variant_object.hpp` -> `variant.hpp` -> `exception.hpp` -> `logger.hpp` -> `log_message.hpp` -> `variant_object.hpp`. That's why no FC_ASSERTS in `variant.hpp`. We could fix this later in a separate PR.